ccmbioinfo/lfs
# CARDS-2496: Update YVM filters and column mapping

To test:
- build this branch with `mvn clean install -Pdocker`
- in `compose-cluster` generate a docker compose file with `python3 generate_compose_yaml.py --mssql --cards_project cards4prems --oak_filesystem --dev_docker_image --composum --adminer --server_address=localhost:8080`
- in `compose-cluster/mssql` run `python3 ./generate_test_YVM_sql.py -n 200 yvm_sample.sql` to generate a sample file
- start with `docker-compose build && docker-compose up`
- at `http://localhost:1435/?mssql=mssql&username=sa&db=master&ns=path&import=` import the sample sql file from `compose-cluster/mssql`
- access `http://localhost:8080/Subjects.importClarity?config=Your%20Voice%20Matters%20-%20PMCC%20outpatient%20visits`  to import the visits
- wait a bit for the import to finish
- check that some visits were imported
- check that we have a collection of visit statuses from the allowed values (Arrived, Completed, Left without seen, HH Incomplete, Scheduled), but no Canceled or No show
- check that only Outpatient and Person events have been imported
- stop and clean up with `docker-compose rm -vf && ./cleanup.sh`
